composer/loggers/mlflow_logger.py

```diff
@@ -437,6 +437,7 @@ def post_close(self):
             import mlflow
 
             assert isinstance(self._run_id, str)
+            mlflow.flush_async_logging()
```

Comment:
Should this be non-async? because this is in termination...?

Reply:
yes, this is a blocking call.